### ISSUE TITLE
Use cases for gene locations. 

### ISSUE THREAD

**Issue by reece** [ga4gh/vrs#263]
Imported from #201: Use cases for Gene locations.

## Use cases
Concrete examples of the symbolic reference use case described by @reece [above](https://github.com/ga4gh/vr-spec/issues/201#issuecomment-678888990).
### Gene-specified fusions
- [BCR / ABL1 fusion](https://civicdb.org/events/genes/4/summary/variants/1/summary)
- [KIAA1549 / BRAF fusion](https://civicdb.org/events/genes/5/summary/variants/618/summary/evidence/7148/summary#evidence)
- [ALK fusions](https://ckb.jax.org/molecularProfile/showMolecularProfileByVariant?geneVariantId=2602)
- [RET fusions](https://www.oncokb.org/gene/RET/Fusions)
### Gene-specified CNAs
- [EGFR amplification](https://civicdb.org/events/genes/19/summary/variants/190/summary)
- [ERBB2 amplification](https://www.oncokb.org/gene/ERBB2/Amplification)
- [PTEN loss](https://civicdb.org/events/genes/41/summary/variants/214/summary)
### Gene-specified expression variants
- [CCND1 overexpression](https://civicdb.org/events/genes/8/summary/variants/20/summary#variant)
- [RRM underexpression](https://civicdb.org/events/genes/5051/summary/variants/1257/summary#variant)
### Gene region-specified alterations
- [EGFR Exon 19 deletion](https://www.oncokb.org/gene/EGFR/Exon%2019%20deletion)
- [KIT Exon 11 mutations](https://ckb.jax.org/molecularProfile/showMolecularProfileByVariant?geneVariantId=19441)
- [FLT3 tyrosine kinase domain mutation](https://civicdb.org/events/genes/24/summary/variants/56/summary#variant)
### Gene wild type variants
- [EGFR wild type](https://civicdb.org/events/genes/19/summary/variants/2174/summary#variant)
- [KRAS wild type](https://www.oncokb.org/gene/KRAS/Wildtype)
- [APC wild type](https://ckb.jax.org/molecularProfile/showMolecularProfileByVariant?geneVariantId=416)
### Other categorical gene variants
- [FLT3 in-frame ITD](https://civicdb.org/events/genes/24/summary/variants/55/summary#variant)
- [CEBPA mutations](https://civicdb.org/events/genes/15/summary/variants/29/summary#variant)
